awx/sso/conf.py

```diff
@@ -92,1568 +92,1576 @@ def __call__(self):
     ]
 )
 
-###############################################################################
-# AUTHENTICATION BACKENDS DYNAMIC SETTING
-###############################################################################
-
-register(
-    'AUTHENTICATION_BACKENDS',
-    field_class=AuthenticationBackendsField,
-    label=_('Authentication Backends'),
-    help_text=_('List of authentication backends that are enabled based on license features and other authentication settings.'),
-    read_only=True,
-    depends_on=AuthenticationBackendsField.get_all_required_settings(),
-    category=_('Authentication'),
-    category_slug='authentication',
-)
+if settings.DIRECT_SHARED_RESOURCE_MANAGEMENT_ENABLED:
```

Comment:
did we check if this breaks the UI?

Reply:
the external auth settings links are still present, and lead to 404s